puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/deploy_puppet.yaml

```diff
@@ -0,0 +1,16 @@
+- job:
+    name: deploy_puppet
+    project-type: workflow
+    logrotate:
+      numToKeep: 3
+    scm:
+      - foreman-infra
+    triggers:
+      - github
+      - pollscm:
+          cron: '*/5 * * * *'
```

Comment:
The modern standard is ```H/5``` which spreads the jobs around a bit rather than hammering everything at the whole hour.

Reply:
okay cool. i was just going off of what was in the job originally.

Reply:
updated